src/weakptr.c

```diff
@@ -868,6 +960,9 @@ static Int InitKernel (
     InitMarkFuncBags ( T_WPOBJ +COPYING, MarkWeakPointerObj   );
```

Comment:
Completely unrelated stupid question (which does not need to be addressed for merging): why is it possible to have `USE_GASMAN` and `USE_THREADSAFE_COPYING`?